### PR TITLE
Use JsSourceParser as default parser for AstAnalyser class

### PR DESCRIPTION
**Pull request by madina0801**
Issue #223 
It was suspiciously easy, let me know if there are any issues :)

### CODE REVIEW

**Comment by PierreDemailly** [comment]
Thanks for contributing, could you update the type here? https://github.com/NodeSecure/js-x-ray/blob/7422acba002fc75059a016c6e10a29ec575a6437/types/api.d.ts#L81

**Comment by fraxken** [comment]
You can probably write a new test in https://github.com/NodeSecure/js-x-ray/blob/master/test/AstAnalyser.spec.js

If we declare the class without any parser and then use it (it should not throw any error).

File: test/AstAnalyser.spec.js [hunk 1]
```diff
@@ -225,6 +225,18 @@ describe("AstAnalyser", (t) => {
       assert.strictEqual(preparedSource, "\nconst yo = 'foo'\n");
     });
   });
+
+  describe("constructor", () => {
+    it("should not throw an error when instantiated without a custom parser", () => {
+      assert.doesNotThrow(() => {
+        const analyser = getAnalyser();
```
Comment (PierreDemailly) [inline test/AstAnalyser.spec.js]: Actually this test is wrong.
The `getAmalyser()` function returns the `AstAnalyser` instanciated with `JsSourceParser()`.

```js
analyser = new AstAnalyser(new JsSourceParser());
```

```suggestion
        const analyser = new AstAnalyser();
```

**Review by PierreDemailly — CHANGES_REQUESTED** [review]

**Review by PierreDemailly — APPROVED** [review]

**Review by fraxken — APPROVED** [review]

**Comment by fraxken** [comment]
@allcontributors please add @madina0801 for code

**Pull request merged**